content/docs/integrations/kubernetes/index.md

```diff
@@ -10,423 +10,177 @@ integrationType: OSS plugin
+i.e. manually `kubectl edit -n kube-system configmap/aws-auth` or using eksctl `eksctl create iamidentitymapping --cluster <my-cluster> --region=<region-code> \
+--arn arn:aws:iam::<your-account-number>:role/roadie-roadie-read-only-role --username roadie --group system:authenticated`
 
-### Step 3: Set RBAC for new role
-
-1. Edit your Kubernetes aws-auth Configmap as per [the EKS docs](https://docs.aws.amazon.com/eks/latest/userguide/add-user-role.html).
-
-It should look something like this:
+ℹ️ NB: The role arn is the same as the arn you copied from the Output `RoadieIamRoleArn` after running the Cloudformation template in step 1. 
 
+Running `kubectl describe -n kube-system configmap/aws-auth` the new entry should look something like this:
 ``` yaml
  - mapRoles:
    - "groups":
       - "system:authenticated"
-      "rolearn": "[ASSUMABLE ROLE ARN]"
+      "rolearn": "arn:aws:iam::<your-account-number>:role/roadie-roadie-read-only-role"
       "username": "roadie"
 ```
 
-⚠️ In the yaml snippet above, be sure to replace "[ASSUMABLE ROLE ARN]" with the ARN of the assumable role created in step 1.
-
-
-2. Create an RBAC for this user:
-
-``` yaml
----
-apiVersion: rbac.authorization.k8s.io/v1
-kind: ClusterRole
-metadata:
-  name: roadie-assume-role
-rules:
-  - apiGroups: [""]
-    resources:
-      - pods
-      - configmaps
-      - services
-      - deployments
-      - replicasets
-      - horizontalpodautoscalers
-      - ingresses
-    verbs: 
-      - "get"
-      - "list"
-      - "watch"
-  - apiGroups: ["batch"]
-    resources:
-      - jobs
-      - cronjobs
-    verbs: 
-      - "get"
-      - "list"
-      - "watch"
-  - apiGroups: ["extensions", "apps", "autoscaling", "networking.k8s.io"]
-    resources: ["deployments", "ingresses", "replicasets", "horizontalpodautoscalers"]
-    verbs: 
-      - "get"
-      - "list"
-      - "watch"
----
-apiVersion: rbac.authorization.k8s.io/v1
-kind: ClusterRoleBinding
-metadata:
-  name: roadie-assume-role-binding
-roleRef:
-  apiGroup: rbac.authorization.k8s.io
-  kind: ClusterRole
-  name: roadie-assume-role
-subjects:
-  - kind: User
-    name: roadie
-```
-
-3. Add this to your cluster and you should now be good to go!
-
-ℹ️ Note you can reuse the Role if you have multiple clusters. You will have to configure the RBAC though.
-
-### Step 4: Adding a cluster to roadie
+### Step 4: Add the cluster settings in Roadie
 
-1. Navigate to your Kubernetes settings in Roadie and click on add item.
+1. Navigate to your Kubernetes settings in Roadie and click on add item `https://[tenant-name].roadie.so/administration/settings/plugins/kubernetes`
+2. If using the Broker, select Service Account type, add a name of your choice and an API endpoint of `broker://<the-token-from-step-2>`
+3. If not using the Broker, select EKS, add the Cluster's API Server Endpoint, the Cross Account Role ARN and External ID from Step 1.
```

Comment:
They should click the"test role" and "test cluster" buttons.

Reply:
I already added the role and external id in step 1.2. I dont need to do it a second time.. I think possibly we should lead with the settings page... i.e. Step 1 go to the settings page, 2 set a name, 3 set the kube url, 4 client create role, if you havent already, 5 enter the role details, 6 run the helm chart etc